src/bindings/python/pyproject.toml

```diff
@@ -1,3 +1,40 @@
+[project]
```

Comment:
should we remove requirements.txt file at least for OpenVINO, excluding for testing?

Reply:
tried, it is still used in Jenknis CI, lets remove it separately